python/tests/llm/clients/anthropic_bedrock/conftest.py

```diff
@@ -0,0 +1,17 @@
+"""Configuration for AnthropicBedrock tests."""
+
+from collections.abc import Generator
+
+import pytest
+
+from mirascope.llm.clients.anthropic_bedrock import clients
+
+
+@pytest.fixture(autouse=True)
+def clear_client_cache() -> Generator[None, None, None]:
+    """Clear the LRU cache before and after each test to prevent interference."""
+    clients.clear_cache()
```

Comment:
Rather than doing this for every new client, maybe we should have an autouse=True clear_all_client_caches() in the top level conftest?